DAT_files/bd+56d524.dat

```diff
@@ -7,5 +7,9 @@ H = 9.000 +/- 0.030
 K = 8.970 +/- 0.020
 IUE = IUE_Data/bd+56d524_iue.fits
 FUSE = FUSE_Data/bd+56d524_fuse.fits
+SpeX_SXD = SpeX_Data/bd+56d524_SXD_spex.fits
+SpeX_LXD = SpeX_Data/bd+56d524_LXD_spex.fits
 sptype = B1V
 uvsptype = B0.5V
+corfac_spex_SXD = 0.9849831324848323
+corfac_spex_LXD = None
```

Comment:
Is there a reason to include the corfac and set it equal to `None` instead of just not including this corfac and then just not applying it?

Reply:
The corfac is None when it could not be calculated because there was no photometry. I included this to make a distinction between stars for which the calculation has been done (but failed), and stars for which the calculation has not been done yet. If I don't include this line, you don't know whether the calculation has been tried or not, and you might end up doing the same stars over and over again. But that was just my thinking :)

Reply:
Makes sense.  Sounds fine to me.  Thanks.